### ISSUE TITLE
complete addition of PyTorch classes

### ISSUE THREAD

**Issue by realmarcin** [berkeleybop/artificial-intelligence-ontology#35]
Done:

- pooling
- normalization

Todo:
- other layers
- map PyTorch <-> TensorFlow
